### ISSUE TITLE
Maintainers

### ISSUE THREAD

**Issue by TheCarpetMerchant** [google/flutter.widgets#422]
There's a lot of pull requests that are month-old and do not have any feedback/merging. I myself am currently using my own branch in production because of this.

I do not know how or where this conversation is supposed to take place, so I'm posting here and tagging @mit-mit . Feel free to close this as soon as possible, and possibly amend the CONTRIBUTING.md to add a small section about maintainers and what behavior is expected from external users in such cases.

Thanks.

**Comment by jamesderlin** [comment]
The project is a mono-repo for several packages, each with different owners.  Primarily it is the individual package's owner's responsibility to maintain their own package, but they also have other responsibilities, and some of them possibly have moved on to other things.

@xster maybe can provide more clarity.

**Comment by ted-marozzi** [comment]
Agreed, it seems linked scroll controller needs some love. I have been waiting for #304 to land for so long that I had to fork the package. Is it possible that someone can take ownership of this package or could ownership be shared with the community? I would be happy to help maintain it.

**Comment by josephcrowell** [comment]
Please do. It seems effectively abandoned at this point.

**Comment by mit-mit** [comment]
I think the README caoptures the expected support level:

> The product and source code here are neither endorsed by Google nor the Flutter team. They are mainly made available as a reference. Support is best-effort.


But @TheCarpetMerchant has a good point; we should have something similar in CONTRIBUTING.

**Comment by TheCarpetMerchant** [comment]
Thanks Michael. In that case, I'll take the issue to @jonasfj, who seems to be the person being the publishing of these packages on pub.dev under the name [google.dev](https://pub.dev/publishers/google.dev/packages).
Jonas, could you either take up reviewing of opened PRs, or if you cannot at the moment delegate such tasks for this repo to someone else ? That would be much appreciated.

**Comment by mit-mit** [comment]
Just to clarify: you should not expect that all PRs in this repo will be reviewed and merged. As the readme says, _"They are mainly made available as a reference. Support is best-effort."_ But we could certainly make that clearer in CONTRIBUTING so that no one wastes their time.

**Comment by jamesderlin** [comment]
To my knowledge, Jonas has no involvement with any of the packages published by this repo.  Jonas only manages access to the google.dev publisher on pub.dev.  As I mentioned before, you probably should be asking @xster.

**Comment by jonasfj** [comment]
There is a spreadsheet with owners. But yes, I only manage ownership. If owners have left let me know.

**Comment by TheCarpetMerchant** [comment]
@xster would be the owner here, if I understand correctly, and hasn't responded to pings. Does anyone have an alternative way of contacting this person ?

**Comment by josephcrowell** [comment]
Looks like he works at Google. I'm sure @jonasfj  has a way of contacting

**Comment by jonasfj** [comment]
@mehmetf, @tarobins, @polina-c, you the ones I have listed as owners/maintainers of packages in this repository.

It'd be nice if you'd agree on how to handle PRs, maybe write a `CONTRIBUTING.md`. And ideally community what prospective contributors can expect.

You could consider one of the following options:

 * Decide to review all PRs / issues and deal with them.
   * Or at-least establish some sort of process for it.
 * Find a process to leverage community contributions.
   * Maybe, find trusted community contributors you can add to the repository.
   * If you want I'm happy to help configure automated publishing.
   * Or transfer ownership of a package to a community contributor, if you have a good option.
 * Clearly state that contributors should not expect timely feedback on PRs / issues.
   * Assuming you're not interested in community contributions.
   * Being open to contributions is a non-trivial amount of work.
   * It's better to be clear about what contributors can expect.
   * If you don't want contributions, maybe suggest users fork the packages instead.
 * Discontinue all the packages and mark the repository as abandoned.
   * If you've lost all interest that's fair, and communicating it is important.
   * This would encourage users to fork.

I'm sure there are other options and variations too. I think it's important to clearly communicate what prospective contributors can expect. And give them some guidance: should they file a PR or fork? :)

**Comment by TheCarpetMerchant** [comment]
Bumping since we haven't heard from @mehmetf , @tarobins and @polina-c .

**Comment by tarobins** [comment]
Hi, 

Sorry.  I do have it in my goals for this quarter to revive ScrollablePositionedList, after much neglect.  Let me see if I can get that rolling this week.
